pipeline/filters/grep.md

```diff
@@ -47,6 +47,8 @@ $ bin/fluent-bit -i tail -p 'path=lines.txt' -F grep -p 'regex=log aa' -m '*' -o
 
 ### Configuration File
 
+{% tabs %}
+{% tab title="fluent-bit.conf" %}
 ```python
 [INPUT]
```

Comment:
```suggestion
[SERVICE]
    parsers_file /path/to/parsers.conf

[INPUT]
```

Reply:
@lecaros Thank you for reviewing. I added.